packages/cli/package.json

```diff
@@ -31,7 +31,6 @@
     "lint": "eslint --color --ext .ts src/ test/",
     "lint:fix": "yarn run lint --fix",
     "pretest": "yarn run check-types",
-    "test:unit": "nyc --cache-dir .nyc_output/.cache -e .ts mocha 'test/unit/**/*.test.ts'",
```

Comment:
Why remove? This command should work even with no files

Reply:
undone